Tests/SwiftIDEUtilsTest/Assertions.swift

```diff
@@ -0,0 +1,103 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import SwiftIDEUtils
+import SwiftParser
+import SwiftSyntax
+import _SwiftSyntaxTestSupport
+
+func assertClassification(
+  _ source: String,
+  expectedClassificationSpec: [ClassificationSpec] = []
+) {
+  let tree = Parser.parse(source: source)
+  assertClassificationImpl(source: source, classifications: tree.classifications.filter { $0.kind != .none }, expectedClassificationSpec: expectedClassificationSpec)
+}
+
+func assertClassification(
+  _ source: String,
+  in range: ByteSourceRange,
+  expectedClassificationSpec: [ClassificationSpec] = []
+) {
+  let tree = Parser.parse(source: source)
+  assertClassificationImpl(source: source, classifications: tree.classifications.filter { $0.kind != .none && ($0.endOffset > range.offset) && ($0.offset < range.endOffset) }, expectedClassificationSpec: expectedClassificationSpec)
+}
```

Comment:
Could we unify this with `assertClassification` above by having `range` be optional and defaulted to `nil`? Then we also wouldn’t need `assertClassificationImpl` anymore if we get rid of `assertTokensClassification`.